README.md

```diff
@@ -50,6 +50,11 @@ The following scripts are available, for further information please refer to the
 - <b>test</b>: `npm t` - _execute the unit test suite_
 - <b>build</b>: `npm run build` - _bundle the project for production_
 
+### Local database
+
+The project needs a database available in order to run locally. The `npm start` command, and by extension `npm run dev` as well, will automatically set up a
+database running in a docker container for you and populate it with sample data. The connection details for viewing the database can be found in `src/infrastructure/localSecretsManagerService.ts`. Running `npm start` repeatedly will only initialise a single docker container, and your changes to the database will only be removed if you delete the container.
```

Comment:
Put a note in about deleting the container and restarting if there are any problems